File: CurriculumRepository.CORE/Data/Models/Activity/LaDTO.cs
using CurriculumRepository.CORE.Entities;
using System;
using System.Collections.Generic;


namespace CurriculumRepository.CORE.Data.Models.Activity
{
    public class LaDTO
    {
        public LaDTO()
        {
            StrategyMethods = new List<StrategyMethodBM>();
            TeachingAids = new List<TeachingAid>();
        }
        public int Idla { get; set; }
        public int OrdinalNumber { get; set; }
        public string Laperformance { get; set; }
        public string Latype { get; set; }
        public bool DigitalTechnology { get; set; }
        public string Laname { get; set; }
        public TimeSpan Laduration { get; set; }
        public string Ladescription { get; set; }
        public List<StrategyMethodBM> StrategyMethods { get; set; }
        public string Lacollaboration { get; set; }
        public int? Lagrade { get; set; }
        public List<TeachingAid> TeachingAids { get; set; }
        public string Laacknowledgment { get; set; }
    }
}

File: CurriculumRepository.CLIENT/client-app/src/app/models/activity/LaDTO.ts
import { Byte } from '@angular/compiler/src/util';
import { TimeSpan } from 'src/app/helpers/timespan';
import { StrategyMethodBM } from '../StrategyMethodBM';
import { TeachingAid } from '../TeachingAid';

export class LaDTO {
    Idla: number;
    OrdinalNumber: number;
    Laperformance: string;
    Latype: string;
    DigitalTechnology: boolean;
    Laname: string;
    Laduration: TimeSpan;
    Ladescription: string;
    LastrategyMethod: StrategyMethodBM[];
    Lacollaboration: string;
    Lagrade: Byte;
    LateachingAids: TeachingAid[];
    Laacknowledgment: string;
}